apps/condo/domains/acquiring/access/MultiPayment.js

```diff
@@ -8,43 +8,41 @@ const { throwAuthenticationError } = require('@condo/domains/common/utils/apollo
 const { RESIDENT } = require('@condo/domains/user/constants/common')
 
 
+
 async function canReadMultiPayments ({ authentication: { item: user } }) {
     if (!user) return throwAuthenticationError()
     if (user.deletedAt) return false
-    if (user.isAdmin || user.isSupport) return {}
-    const userId = user.id
-    // Resident user can get only it's own MultiPayments
+
+    if (user.isSupport || user.isAdmin) return {}
+
     if (user.type === RESIDENT) {
         return {
-            user: { id: userId },
+            user: { id: user.id },
         }
     }
+
     // Acquiring integration account can get only MultiPayments linked to this integration
-    return {
-        integration: { accessRights_some: { user: { id: userId } } },
-    }
+    return { integration: { accessRights_some: { user: { id: user.id } }, deletedAt: null } }
 }
 
 async function canManageMultiPayments ({ authentication: { item: user }, operation, itemId }) {
     if (!user) return throwAuthenticationError()
     if (user.deletedAt) return false
+
     if (user.isAdmin) return true
     // Can be created only through custom mutation or by admin, can be modified by acquiring integration account
-    if (operation === 'create') {
-        return false
-    } else if (operation === 'update') {
-        if (!itemId) return false
-        // Acquiring integration account can update only it's own multipayment
-        return { integration: { accessRights_some: { user: { id: user.id } } } }
-    }
+    if (operation === 'create') return false
+    // Acquiring integration account can update only it's own multipayment
+    if (operation === 'update' && itemId) return { integration: { accessRights_some: { user: { id: user.id } } } }
+
     return false
 }
 
 async function canReadMultiPaymentsSensitiveData ({ authentication: { item: user }, existingItem }) {
-    if (user.deletedAt) return false
+    if (!user || user.deletedAt) return false
```

Comment:
There's no chance to get there without user, since list-level checks are before field-level, where we're already throwing an error...

But, there's difference in behaviour (I guess) if my request is something like `{ A { B { ... } } }` ,
where A is model which unauthorised user have access, and B is auth-required relation.
So in you case we will throw an error, in mine we're just blocking access, so result will be like `{ A { null } }`